qemu/tests/hugepage_specify_node.py

```diff
@@ -84,3 +96,4 @@ def run(test, params, env):
         finally:
             vm.destroy()
             hp_config.cleanup()
+            time.sleep(10)
```

Comment:
Could you explain the reason for adding this line ? Thanks!

Reply:
For stability.
When we allocate 1GB or 16GB huge page, It needs to be allocated from CMA (Contiguous Memory Allocator).
CMA works depend on memory fragmentation and migration.
We did this step too quickly in the automation loop.